cadasta/test/resource_tests/test_location_resources.py

```diff
@@ -1,18 +1,16 @@
 import os
-from selenium_tests.test import SeleniumTestCase
-from selenium_tests.webdriver import CustomWebDriver
+
 from selenium.webdriver.common.action_chains import ActionChains
 from selenium.webdriver.support.ui import Select
-from selenium_tests.pages import ProjectsPage
-from selenium_tests.entities import Project
 
+from ..base_test import SeleniumTestCase
+from ..entities import Project
+from ..pages import ProjectsPage
 
-class AddLocationResource(SeleniumTestCase):
 
-    def setUp(self):
-        self.wd = CustomWebDriver()
+class AddLocationResource(SeleniumTestCase):
 
-    def test_attach_existing_resource_to_new_location(self):
+    def xtest_attach_existing_resource_to_new_location(self):
```

Comment:
Yes. These tests correspond to the tests that were commented out in the original **cadasta_test_suites.py** manual list of tests. I assumed that they were commented out because they were not yet working properly. So I renamed the methods so that pytest doesn't auto-discover and run them. I will eventually go over and check why these tests were commented out and try to debug them.